NuGet.config

```diff
@@ -8,6 +8,7 @@
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
     <add key="dotnet8" value="https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet8/nuget/v3/index.json" />
     <add key="dotnet8-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8-transport/nuget/v3/index.json" />
+    <add key="BuildXL" value="https://pkgs.dev.azure.com/ms/BuildXL/_packaging/BuildXL/nuget/v3/index.json" />
```

Comment:
It's actually open source: https://github.com/microsoft/BuildXL

Reply:
I have no opinion on this.  It doesn't matter to source-build since the packages are conditionally referenced only for netFX.  Source-build dynamically removes all feeds when it builds and dynamically injects the source-built feeds.  Source build must build everything from source so it is only valid to reference the built packages.